dev/tools/controllerbuilder/template/apis/types.go

```diff
@@ -83,6 +84,7 @@ type {{ .Kind }} struct {
 	metav1.TypeMeta   ` + "`" + `json:",inline"` + "`" + `
 	metav1.ObjectMeta ` + "`" + `json:"metadata,omitempty"` + "`" + `
 
+	// +required
```

Comment:
I think this is fine to do because we expected to have the common case to always want to require spec